Source/WebCore/page/Performance.h

```diff
@@ -34,11 +34,13 @@
 
 #include "ContextDestructionObserver.h"
 #include "DOMHighResTimeStamp.h"
+#include "EventNames.h"
```

Comment:
I think you should be able to forward declare EventType instead (maybe I'm wrong though) and only import in the CPP